alica_engine/src/engine/BasicPlan.cpp

```diff
@@ -14,6 +14,12 @@ BasicPlan::BasicPlan(IAlicaWorldModel* wm)
 {
 }
 
+const std::unordered_map<int64_t, std::unordered_set<int64_t>>& BasicPlan::getApplicationEntrypointContext(std::vector<int64_t> entryPointIds)
+{
+    std::unordered_map<int64_t, std::unordered_set<int64_t>> map;
+    return map;
```

Comment:
after the signature of the method is changed to 
`virtual bool getApplicationEntrypointContext(std::unordered_map<int64_t, std::unordered_set<int64_t>& entryPointMap) override;` the default implementation would just return entryPointMap as is

Reply:
also add comments here saying that:
- this method is called before task assignment & therefore the onInit() method would not have been called yet
- explaining the input & outputs: there an entry for each entry point in the unordered_map & the unordered set denotes the dyanmic entyrpoint for each entry point
- return value true signifies that EntryPointMap changed since last invocation of this method